stock_available/product.py

```diff
@@ -0,0 +1,66 @@
+#-*- coding: utf-8 -*-
```

Comment:
adding a space after `#` will make flake8 happy and travis :green_heart: :

    ./stock_available/product.py:1:1: E265 block comment should start with '# '